components/gc-featured-link/gc-featured-link-en.html

```diff
@@ -1,116 +1,96 @@
 ---
-breadcrumb: [
-	{ "title": "GCWeb home", "link": "https://wet-boew.github.io/GCWeb/home.html" }
-]
-dateModified: "2021-06-03"
-description: "Documentation and working example for the featured link - provisional feature"
-language: "en"
-altLangPage: "gc-featured-link-fr.html"
-title: "Featured link - Provisional feature"
+title: Featured link - Provisional feature
+description: Documentation and working example for the featured link - provisional feature
+language: en
+altLangPage: gc-featured-link-fr.html
+dateModified: 2021-07-02
+layout: no-container
 ---
-{% include alert-provisional.html siteroot="../../" %}
+<div class=:"container">
+	<h1>{{ page.title }}</h1>
 
-<span class="wb-prettify all-pre hide"></span>
+	{% include alert-provisional.html siteroot="../../" %}
 
-<dl class="horizontal">
-	<dt>Status</dt>
-	<dd>Provisional</dd>
-	<dt>Type</dt>
-	<dd>Canada.ca design pattern</dd>
-	<dt>Last review</dt>
-	<dd>2021-06-02</dd>
-	<dt>Conforming to</dt>
-	<dd>Content and IA Specification - Version 2.1.2</dd>
-	<dt>Guidance</dt>
-	<dd>n/a</dd>
-	<dt>Technical name</dt>
-	<dd><code>gc-featured-link</code></dd>
-</dl>
+	<span class="wb-prettify all-pre hide"></span>
 
-<h2>Purpose</h2>
-<p>This adds a featured link to a page that your institution needs to feature prominently. It should be used to bring attention to warnings or notices.</p>
+	<dl class="horizontal">
+		<dt>Status</dt>
+		<dd>Provisional</dd>
+		<dt>Type</dt>
+		<dd>Canada.ca design pattern</dd>
+		<dt>Last review</dt>
+		<dd>2021-07-02</dd>
+		<dt>Conforming to</dt>
+		<dd>Content and IA Specification - Version 2.1.2</dd>
+		<dt>Guidance</dt>
+		<dd>n/a</dd>
+		<dt>Technical name</dt>
+		<dd><code>gc-featured-link</code></dd>
+	</dl>
 
-<h2>Working examples</h2>
+	<h2>Purpose</h2>
+	<p>This adds a featured link to a page that your institution needs to feature prominently. It should be used to bring attention to warnings or notices.</p>
 
-<h3>Featured link (provisional v1.0)</h3>
-<div class="provisional gc-featured-link">
+	<h2>Working examples</h2>
+
+	<h3>Featured link (provisional v1.0)</h3>
+</div>
+
+<section class="provisional gc-featured-link">
```

Comment:
Add a note at the top or before this example explaining the heading sequence is not followed for the demo purpose. Be creative.